packages/manager/apps/sign-up/src/constants.js

```diff
@@ -23,6 +23,11 @@ export const HOME_PAGE = {
   WS: 'https://www.ovh.com/us/es/',
 };
 
+export const SUBSIDIARIES_LABEL_SUFFIX = {
+  IN: '_india',
+  DEFAULT: '',
+};
```

Comment:
**nitpick:** This implementation is not scalable. Lets say if we want to display the same message for some other sub, we will be in a position to copy-paste the existing trans for other sub.